.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
```

Comment:
I think we want to run GH actions on every PR and any branch. No reason not to as far as I'm aware.

```suggestion
on: [push, pull_request]
```

Reply:
If we specify both and resque commiters creates new branch on resque/resque repo, both push and pull_request CI run.
Is it ok?

![image](https://user-images.githubusercontent.com/1796864/124525524-91c4df00-de3a-11eb-9fd1-1280c20a9407.png)

Reply:
Ah, wasn't aware that the build is run twice in that case. I think it's helpful to developers to run CI against their branch before opening a PR, but probably makes sense to use the default here instead. Thanks for calling this out—feel free to change it back.